conventions.md

```diff
@@ -0,0 +1,301 @@
+
+Where a protocol defines metadata that forms part of the message sent via middleware it should be defined as part of the header schema (potentially as MessageTraits), and where the protocol defines data (or transmits metadata in the payload) it should be defined as part of the payload. Note that the payload does not support MessageTraits. 
+
+Tooling to validate that messages match the schema will expect to verify against the schema of the message payload and headers, putting message metadata into a Message Binding makes this harder as tooling must know to look there as well. 
+
+Implementers of producers or consumers are also likely to assume that the the can refer to #/Components/Schema to understand payloads and headers, and adding metadata into bindings in addition makes it harder to comprehend the schema of a message.
+
+A [Message Binding](https://github.com/asyncapi/spec/blob/master/spec/asyncapi.md#messageBindingsObject) should be used to define metadata about the protocol's message format that is used for [Infrastructure as Code](#item-3-support-infrastructure-as-code) or [Scaffolding](#item-4-support-contract-first--scaffolding). Don't use the Message Bindings to describe what is sent; use the Message Binding to describe how it is sent, if required.
+
+```yml
+channels:
+    quux:
+        subscribe:
+            summary: Messages about quux
+            operationId: sendQuux
+            $ref: "#/components/messages/quuxEvent"
+
+...
+
+components:
+    schemas:
+       quux:
+	 type: object
+	 properties:
+	     quuxDetails: 
+	     	type: string	 
+       amqpBasicProperties:
+               type: object
+	       properties:
+                   content_encoding:
+		      type: string
+		   delivery-mode:
+		      type: integer
+		      minimum: 0
+		      maximum: 1
+		   priority:
+		      type: integer
+		      minimum: 0
+		      maximum: 9
+		   correlation-id:
+		      type: string
+		   reply-to:
+		      type: string
+		   expiration:
+		      type: string
+		   message-id:
+		      type: string
+		   timestamp:
+		      type: string
+		   type:
+		       type: string
+		   user-id:
+		       type: string
+		   app-id:
+		       type: string
+       
+    messages:
+        quuxEvent:
+      	    summary: Raised when a quux occurs.
+            description: When a quux happens, describes the quux		  
+      contentType: application/json
+      payload:
+        $ref: "#/components/schemas/quux"
+      traits:
+        $ref: "#/components/schemas/amqpBasicProperties" 
+
+
+```
+
+### **Item 10** Be Consistent with Other Protocols
+Design your binding with the possibility that a producer might expose a message over different bindings - so as to support a wider range of consumers. By following the guidelines in this document your binding should be compatible with other bindings, and simplify the development of tooling that supports those because they operate at the same level.
```

Comment:
> your binding should be compatible with other bindings

How exactly can someone accomplish this? We don't have a common structure among bindings yet.